### PR TITLE
Feature/data loader

### PR DESCRIPTION
**Pull request by edsoncavalcanti**
DataLoader creation

### CODE REVIEW

File: covid_vision/data_loader/data_loader.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            names_noncovid = list_files("data/Dataset COVID-19 Augmented/Non-COVID-19")
+        elif self.data_type == "ultrassound":
+            raise NotImplementedError
+        elif self.data_type == "computedtomography":
+            raise NotImplementedError
+        elif self.data_type == "anamnesis":
+            raise NotImplementedError
+
+        for name in names_covid:
+            self.images['covid'].append(cv2.imread(name))
```
Comment (RaulMedeiros) [inline covid_vision/data_loader/data_loader.py:34]:
```suggestion
            self.images['covid'] = [cv2.imread(name) for name in names_covid]
```

File: covid_vision/data_loader/data_loader.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        for name in names_noncovid:
+            self.images['non-covid'].append(cv2.imread(name))
```
Comment (RaulMedeiros) [inline covid_vision/data_loader/data_loader.py:36]:
```suggestion
        self.images['non-covid'] = [cv2.imread(name) for name in names_noncovid]
```

File: covid_vision/data_loader/data_loader.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
+from covid_vision.utils.file_manipulation import list_files, download_from_google_drive
+import cv2
+
+
+class DataLoader:
+
+    def __init__(self, data_type):
+        self.images = dict()
+        self.images['covid'] = []
+        self.images['non-covid'] = []
```
Comment (RaulMedeiros) [inline covid_vision/data_loader/data_loader.py:10]: Please consider removing:
```suggestion
```

Comment (edsoncavalcanti) [reply covid_vision/data_loader/data_loader.py:10]: done

File: covid_vision/data_loader/data_loader.py (hunk 2 above)
Comment (edsoncavalcanti) [reply covid_vision/data_loader/data_loader.py:36]: done

File: covid_vision/data_loader/data_loader.py (hunk 1 above)
Comment (edsoncavalcanti) [reply covid_vision/data_loader/data_loader.py:34]: done

**Pull request merged**